.golangci.yaml

```diff
@@ -42,17 +42,18 @@ run:
   #  - ".*\\.my\\.go$"
   #  - lib/bad.go
 
+  # Define the Go version limit.
+  # Mainly related to generics support since go1.18.
+  # Default: use Go version from the go.mod file, fallback on the env var `GOVERSION`, fallback on 1.17
+  go: "1.20"
```

Comment:
follow-up question: should we remove this config completely and rely on the functionality that the version is taken from the `go.mod` file?

Reply:
cc @tklauser

Reply:
I would delete it and see what happens.

Reply:
Thanks :rocket: i removed it completely.